test/positivity.lean

```diff
@@ -23,6 +23,9 @@ example : 0 ≤ 3 := by positivity
 
 example : 0 < 3 := by positivity
 
+example : (0 : ℝ≥0∞) < 1 := by positivity
+example : (0 : ℝ≥0∞) < 2 := by positivity
```

Comment:
Could you please add a test like this?
```
example : (0 : ℝ≥0∞) < 2 + 2 := by positivity
```
(since the `norm_num` positivity extension should handle that too.)

Reply:
It doesn't work. I suspect it's because `norm_num` doesn't know about truncated subtraction.

Is your "should" meant as in "it should work" or as in "you should make it work"? I think it's pretty out of scope for this PR regardless of your answer.

In fact, my motivation here is to have `positivity` prove that the L2 norm of a nonzero function is positive, and there I really don't have any truncated subtraction